modules/afsmtp/afsmtp.c

```diff
@@ -536,6 +551,31 @@ afsmtp_worker_thread_deinit(LogThrDestDriver *d)
  * Main thread
  */
 
+ static void
+ afsmtp_dd_init_recipient(AFSMTPRecipient *rcpt, GlobalConfig *cfg)
+ {
+   if (!rcpt->addr_tmpl)
+    {
+      gchar *hdr;
+      switch (rcpt->type)
+	{
+	case AFSMTP_RCPT_TYPE_TO:
+	  hdr = "To";
+	  break;
+	case AFSMTP_RCPT_TYPE_CC:
+	  hdr = "Cc";
+	  break;
+	case AFSMTP_RCPT_TYPE_REPLY_TO:
+	  hdr = "Reply-To";
+	  break;
+	default:
+	  return;
+	}
+      rcpt->addr_tmpl = log_template_new(cfg, hdr);
+      log_template_compile(rcpt->addr_tmpl, rcpt->address, NULL);
```

Comment:
Template name, that field is normally used by named templates in the config and not otherwise. Can be null.